src/sonic-config-engine/minigraph.py

```diff
@@ -729,6 +731,21 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
 
+    # Filter ports from mirroring ACL tables that are not active.
+    for acl_table, group_params in acls.iteritems():
+        group_type = group_params.get('type', None)
+
+        if group_type != 'MIRROR' and group_type != 'MIRRORV6':
+            continue
+
+        # We assume that if a LAG was included in the list of ports earlier then it will be active.
```

Comment:
i am not sure I understand the comment here.

Reply:
The acls object gets populated with data from the minigraph in an earlier step. At this point, that object contains every front panel port, unless it is in a port channel, in which case the port channel is included instead. We make the assumption here that if there is a port channel in the acls object then that port channel will be an active port and should not be filtered out of the MIRROR tables.